packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_en_GB.json

```diff
@@ -1,18 +1,18 @@
 {
-  "pci_database_add_title": "Create a database instance",
-  "pci_database_add_description1": "Create a database available in your Public Cloud project. If you would like to use a private network that does not yet exist, please create",
+  "pci_database_add_title": "Create a database service",
+  "pci_database_add_description1": "Create a database service available in your Public Cloud project. If you would like to use a private network that does not yet exist, please create",
   "pci_database_add_description2": "a private network or subnet",
   "pci_database_add_description3": "first.",
-  "pci_database_add_step1_title": "Choose your database",
-  "pci_database_add_step1_description": "Select a database to suit your needs",
-  "pci_database_add_step2_title": "Select a solution",
+  "pci_database_add_step1_title": "Choose your database service",
```

Comment:
For translations, we only add/edit text for FR (French). The translations for the rest of the languages are to be generated via a translation request